query-engine/query-engine-wasm/build.sh

```diff
@@ -45,14 +44,15 @@ fi
 if [ "$WASM_BUILD_PROFILE" = "dev" ]; then
     WASM_TARGET_SUBDIR="debug"
 else
-    WASM_TARGET_SUBDIR="release"
+    WASM_TARGET_SUBDIR="$WASM_BUILD_PROFILE"
```

Comment:
This was just a bug in the code before. The `profiling` profile outputs at said directory.